rxnetty/src/main/java/io/reactivex/netty/protocol/http/UnicastContentSubject.java

```diff
@@ -273,7 +273,7 @@ public void onNext(T t) {
         ReferenceCountUtil.retain(t); // Retain so that post-buffer, the ByteBuf does not get released. Release will be done after reading from the subject.
         state.bufferedSubject.onNext(t);
 
-        if (state.casTimeoutScheduled()) {// Schedule timeout once.
+        if (state.casTimeoutScheduled() && state.state == 0) {// Schedule timeout once and when not subscribed yet.
```

Comment:
I think it's safer to do `state.state == STATES.UNSUBSCRIBED.ordinal()`

This smells like a race (as state can be changed concurrently) but the state does not change from SUBSCRIBED/DISPOSED to UNSUBSCRIBE so we are safe.